src/tools/pbs_python.c

```diff
@@ -2310,11 +2310,37 @@ main(int argc, char *argv[], char *envp[])
 		char *largv[3];
 		int ll;
 		char *pc, *pc2;
```

Comment:
Hi @bhroam ,
       As you have mentioned above i have left pc and pc2 uninitialized, as i am using them only
after strstr and strdup system calls. If there is a failure the calls would update the variables to 
NULL accordingly.

Thanks,
Rampranesh M